src/lib/stitching/gravity/schema.ts

```diff
@@ -12,7 +12,7 @@ import config from "config"
 
 const rootFieldsAllowList = ["agreement"].concat(
   config.USE_UNSTITCHED_VIEWING_ROOM_SCHEMA
-    ? []
+    ? ["viewingRooms"]
```

Comment:
I wanted to remove this field completely ([gravity PR](https://github.com/artsy/gravity/pull/18380)), but seems like it is still used by some clients. Don't know by whom yet. Do I understand it right that it shouldn't be a problem to temporarily keep it here?

Reply:
Sure. Seems like an easy enough field to add directly to MP w/o stitching, esp considering everything else already added! and then you can remove it from here too.